src/asm/fstack.c

```diff
@@ -133,6 +140,25 @@ void fstk_AddIncludePath(char const *path)
 	includePaths[nbIncPaths++] = str;
 }
 
+void fstk_AddIncludeFilePath(char const *path)
+{
+	struct IncludeFile *file = malloc(sizeof(*file));
+
+	if (!file)
+		fatalerror("Failed to allocate memory for included file: %s\n", strerror(errno));
+	file->path = path;
+	file->next = NULL;
+
+	struct IncludeFile *lastFile = includeFiles;
+
+	while (lastFile && lastFile->next)
+		lastFile = lastFile->next;
+	if (lastFile)
+		lastFile->next = file;
+	else
+		includeFiles = file;
```

Comment:
Too far? (Note that `(lastFile ? lastFile->next : includeFiles) = file;` would be valid in C++ but not in C.)
```suggestion
	*(lastFile ? &lastFile->next : &includeFiles) = file;
```

Reply:
I find the C++ way acceptable, but I'm against using pointers for this.